### PR TITLE
docs: add tokens handling [skip chromatic]

### LINKED ISSUE #531: chore: 🔧 switch Tokens Studio default branch to main
## User Story
As a maintainer of the Solid Design System, I would like to get fast updates from Figma tokens, when they were created and approved.

### Suggested Solution
Currently design merges to an own outdated `figma-tokens` branch in the solid repo. This setting has to be changed to the protected `main` branch.

This offers the possibility to create new feature branches (in Figma) for token updates using the `main` branch as basis. 
This creates a PR in Figma to the `main` branch and a respective PR in GitHub automatically.
Therewith the development can review the changes in a PR to the `main` directly and no update from the `figma-tokens` towards the `main` branch is necessary anymore. 

## Technical Information

Image

## Tasks
- [x] Update the `main` branch from the `figma-tokens` branch => NOTE: even folder structure needs to be updated => copy paste to `main` branch might be necessary.
- [x] In Figma the `main` branch needs to be set as basis
- [ ] A detailed explanation and description about the PR title naming convention according to the [development standards](https://github.com/solid-design-system/solid/blob/main/CONTRIBUTING.md#commit-messages) needs to be added to Figma
- [x] `figma-tokens` branch needs to be deleted in the solid repo

## DoR
- [x] Item has business value
- [x] Item has been estimated by the team
- [x] Item is clear and well-defined
- [x] Item dependencies have been identified

## DoD
- [ ] Documentation has been created/updated (if applicable)
- [ ] Implementation works successfully on `feature` branch
- [x] Onboarding for Figma maintainers has been updated

### PR DESCRIPTION
**Pull request by van-nguyen-ht**
## Description:
Adding tokens handling for maintainers in Figma to README
closes #531 

## Definition of Reviewable:
*PR notes: Irrelevant elements should be removed.*
- [x] Documentation is created/updated
- [x] Migration Guide is created/updated
- [x] relevant tickets are linked
- [x] PR is assigned to project board

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -48,6 +48,24 @@ $ cd packages/components
 
 We welcome contributions from the community! If you'd like to contribute to any of the packages or improve the Solid Design System, please follow our [contribution guidelines](CONTRIBUTING.md) provided in this repository.
 
+### Handling tokens (for maintainers)
+
+Any changes in the token set in Token Studio plugin must be in a new branch.
```
Comment (mariohamann) [inline README.md:53]:
```suggestion
Any changes in the token set in Token Studio plugin must be in a new branch branched from `main`.
```

File: README.md [hunk 2]
```diff
@@ -48,6 +48,24 @@ $ cd packages/components
...
+
+![Screen Shot 2024-01-19 at 11 22 24](https://github.com/solid-design-system/solid/assets/118520877/4b7ce66c-a5c7-44f3-86d4-2ae0ab4a902b)
+
+![Screen Shot 2024-01-19 at 11 23 08](https://github.com/solid-design-system/solid/assets/118520877/82e1fd88-ce70-4de9-931b-764e228f0d22)
+
+Note: If the token is related to Figma native style i.e.: color, choose "create styles" so Figma style and token are in sync
```
Comment (mariohamann) [inline README.md:59]:
```suggestion
> Note: If the token is related to Figma native style i.e.: color, choose "create styles" so Figma style and token are in sync
```

File: README.md [hunk 3]
```diff
@@ -48,6 +48,24 @@ $ cd packages/components
...
+
+Then push with a commit message according to the development standards and create a PR ticket to hand over
+
+![Screen Shot 2024-01-19 at 11 28 40](https://github.com/solid-design-system/solid/assets/118520877/6f7f1ca0-6b96-4ff6-bcab-cb66d9ef3adc)
+
+![Screen Shot 2024-01-19 at 11 41 01](https://github.com/solid-design-system/solid/assets/118520877/56a27bce-5cd1-40ee-85b8-731196a66b66)
+
+After front-end's aproval, check if all tests are passed (quality gate, etc.), then hit Squash and Merge button, write a message (especially if there's any changes in token naming or deprecating note "BREAKING CHANGE: ..."
```
Comment (mariohamann) [inline README.md:67]:
```suggestion
After front-end's approval and all tests are passed (quality gate, Chromatic tests etc.) hit `Squash and Merge` button, write a message (especially if there's any changes in token naming or deprecating note "BREAKING CHANGE: ..."
```

**Review by mariohamann — CHANGES_REQUESTED** [review]
I don't think that the README as the first entrypoint on GitHub is the right place for that. This feels more like something that should be added to `Contributing.md`. If you add it there, please align with the writing style we have there.

**Comment by mariohamann** [comment]
> I don't think that the README as the first entrypoint on GitHub is the right place for that. This feels more like something that should be added to `Contributing.md`. If you add it there, please align with the writing style we have there.

@karlbaumhauer @yoezlem What's your opinion on that? This feels way to specific for a general README.

File: README.md [hunk 4]
```diff
@@ -48,6 +48,24 @@ $ cd packages/components
 
 We welcome contributions from the community! If you'd like to contribute to any of the packages or improve the Solid Design System, please follow our [contribution guidelines](CONTRIBUTING.md) provided in this repository.
 
+### Handling tokens (for maintainers)
```
Comment (karlbaumhauer) [inline README.md:51]:
```suggestion
### Handling tokens (for Figma maintainers ONLY)
```

**Comment by karlbaumhauer** [comment]
> > I don't think that the README as the first entrypoint on GitHub is the right place for that. This feels more like something that should be added to `Contributing.md`. If you add it there, please align with the writing style we have there.
> 
> @karlbaumhauer @yoezlem What's your opinion on that? This feels way to specific for a general README.

I personally feel like the README as this is very special for the designers and thy can easily access the information once they are on GItHub (which they need to maintain the tokens). I would suggest to go with the README for now and see if we might get more details for designers and then open up a new file/location for this.

**Review by karlbaumhauer — CHANGES_REQUESTED** [review]

**Comment by karlbaumhauer** [comment]
> > > I don't think that the README as the first entrypoint on GitHub is the right place for that. This feels more like something that should be added to `Contributing.md`. If you add it there, please align with the writing style we have there.
> > 
> > 
> > @karlbaumhauer @yoezlem What's your opinion on that? This feels way to specific for a general README.
> 
> @mariohamann I personally feel like the README as this is very special for the designers and thy can easily access the information once they are on GItHub (which they need to maintain the tokens). I would suggest to go with the README for now and see if we might get more details for designers and then open up a new file/location for this.

agreed in the daily:
it should be part of the `packages/tokens/README.md`
FYI @van-nguyen-ht

**Review by karlbaumhauer — APPROVED** [review]

**Pull request merged**